scripts/trivia.js

```diff
@@ -2158,6 +2191,10 @@ addAdminCommand(["search"], function (src, commandData, channel) {
 }, "Allows you to search through the questions, format /search [query]. Only matches whole words.");
 
 addAdminCommand(["apropos"], function (src, commandData, channel) {
+    if (trivData.eventFlag) {
+        Trivia.sendPM(src, "You cannot use /apropos during event games!", channel);
```

Comment:
IMO grab their command data and blast it over VR to expose the cheaters!! (you don't really have to do this. its fine the way you have it)